doc/source/data/performance-tips.rst

```diff
@@ -187,5 +187,19 @@ To enable deterministic execution, set the preceding to True. This setting may d
 Monitoring your application
 ---------------------------
 
-View the Ray Dashboard to monitor your application and troubleshoot issues. To learn
-more about the Ray dashboard, see :ref:`Ray Dashboard <observability-getting-started>`.
+View the Ray Data dashboard located in the Metrics tab of the Ray Dashboard to monitor your application and troubleshoot issues. Ray Data emits Prometheus metrics in real-time while a Dataset is executing, and the Ray Data dashboard displays these metrics grouped by Dataset. Datasets can also be assigned a name using :meth:`~ray.data.Dataset._set_name`, which prefixes the dataset ID for a more identifiable label.
```

Comment:
looks like it just shows `_set_name` on the UI, can we change it to `Dataset._set_name`?

<img width="965" alt="Screen Shot 2023-10-20 at 11 43 20 AM" src="https://github.com/ray-project/ray/assets/4629931/ea3a5712-3d0e-42ca-9aa5-db7ef9a09eed">

Reply:
>located in the Metrics tab of the Ray Dashboard

Can we add a link to https://docs.ray.io/en/latest/ray-observability/getting-started.html#metrics-view ? So users are clear what we are referring to.